### PR TITLE
tests: bump docker image conformance tests to 1.8.2

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

**Comment by cpanato** [comment]
Retest this please

File: Jenkinsfile [hunk 1]
```diff
@@ -48,7 +48,7 @@ tectonic_smoke_test_env_image = 'quay.io/coreos/tectonic-smoke-test-env:v5.8'
 pipeline {
   agent none
   environment {
-    KUBE_CONFORMANCE_IMAGE = 'quay.io/coreos/kube-conformance:v1.7.5_coreos.0_golang1.9.1'
+    KUBE_CONFORMANCE_IMAGE = 'quay.io/coreos/kube-conformance:v1.8.2_coreos.0_golang1.9.1'
```
Comment (mxinden) [inline Jenkinsfile]: From my side we don't have to carry on adding the `.0_golang1.9.1` tag to the conformance test image. I just did so, to update the already existing `v1.7.5_coreos` without removing the old one.

Comment (cpanato) [reply Jenkinsfile]: done

**Comment by cpanato** [comment]
Timeout during conformance test execution was above 1.5 we will observe if still continue we will bump the time. merging this pr

**Pull request merged**